### PR TITLE
@rooks/use-countdown

### PR DESCRIPTION
**Pull request by Stupidism**
# Description
- Add a hook to do countdown

image
## Snapshot 

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist for Author
- [x] Tests written where needed
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "@babel/plugin-proposal-export-default-from": "7.5.2",
     "@babel/preset-env": "7.5.5",
     "@babel/preset-react": "7.0.0",
-    "@testing-library/react": "8.0.1",
+    "@testing-library/react": "8.0.9",
+    "@testing-library/react-hooks": "3.2.1",
```
Comment (Stupidism) [inline package.json]: I used another testing-library for hooks so we don't need to write mock components to represent the results of hooks any more.

@testing-library/react-hooks used a feature in react@^16.9.
It doesn't affect the hooks themselves but only needs react to be over 16.9 in the dev environment.
I'm not very sure if this is a good way to do it.

**Comment by Stupidism** [comment]
@imbhargav5 Do you have time to review this PR?

**Comment by imbhargav5** [comment]
Yep will take a look tomorrow. Thanks!

**Comment by imbhargav5** [comment]
I really like it on first look. Let me merge during the weekend.

**Comment by imbhargav5** [comment]
:fire: Thank you for this really awesome hook! @Stupidism

**Comment by Stupidism** [comment]
Thank you for merging it~~~ Would love to be a contributor to the hooks world~~~

**Pull request merged**